sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -44,16 +44,16 @@ import org.apache.spark.unsafe.types.{ByteArray, UTF8String}
 // This file defines expressions for string operations.
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 
-
```

Comment:
I think we need to revert the changes in this file?

Reply:
Thanks!